jobs/label_misinformation/app/s3_utils.py

```diff
@@ -32,12 +34,13 @@ def get_bucket_key(date, channel, filename: str = "*", suffix: str = "parquet")
     return f"year={year}/month={month:1}/day={day:1}/channel={channel}/{filename}.{suffix}"
 
 
-def get_bucket_key_folder(date, channel, root_folder=None) -> str:
+def get_bucket_key_folder(date, channel, root_folder=None, country:Union[Country, CountryCollection]=FRANCE_COUNTRY) -> str:
     (year, month, day) = (date.year, date.month, date.day)
     key = f"year={year}/month={month:1}/day={day:1}/channel={channel}/"
+    if country not in LEGACY_COUNTRIES:
+        key = f"country={country.name}/" + key
```

Comment:
too quick review for me: will CountryCollection work here ?

I would suggest to use only one type for a parameter, what do you think ?